streams/src/main/java/org/apache/kafka/streams/processor/internals/Tasks.java

```diff
@@ -273,6 +274,12 @@ Collection<Task> allTasks() {
         return readOnlyTasks;
     }
 
+    Collection<Task> notPausedTasks() {
```

Comment:
Unit tests would be great!

Reply:
@cadonna Same deal here, I started to try and test `Tasks` instances and I started down a path of mocking quite a bit of the internal classes.

I think I'm either missing an easier approach or testing these functions directly may require a decent amount of effort.